apps/fxc-front/src/app/logic/track.ts

```diff
@@ -46,3 +58,94 @@ async function fetchAndReturnGroupIds(url: string, options?: RequestInit): Promi
   }
   return Array.from(groupIds);
 }
+
+export function scoreTrack(track: RuntimeTrack){
+	const request: ScoreRequest = {track: track, league: currentLeague(store.getState())}
```

Comment:
`{track: track}` could be simplified as `{track}` in modern JS/TS.

I would inject `currentLeague(store.getState())` so that score Track is more self contained and does not depend on the store - would help with testing.

Reply:
injected the league in function parameter